packages/database/src/modelClasses/User.js

```diff
@@ -38,6 +38,19 @@ export class UserModel extends DatabaseModel {
     return UserType;
   }
 
+  /**
+   * Returns the user that is used for submitting surveys when not logged in
+   * @returns {Promise<null|*>}
+   */
+  async findPublicUser() {
+    const user = await this.findOne({ email: 'public@tupaia.org' });
```

Comment:
I reckon we should have the public user as a constant somewhere common, so we can refer to it that way